docs/fundamentals/blsync.md

```diff
@@ -0,0 +1,75 @@
+![Finding a suitable epoch](/images/docs/blsync1.png)
+- Open the epoch details and find the first proposed slot at the end of the page.
+![Finding the first slot](/images/docs/blsync2.png)
+- Copy the block root field.
+![Copy the block root](/images/docs/blsync3.png)
+
+#### Example
+
+The following command can be used to start Geth with blsync on the Sepolia network. Note that the checkpoint root will be outdated two weeks after the writing of this page and a recent one will have to be picked according to the guide above:
+
+```terminal
+./build/bin/geth --sepolia --beacon.api https://sepolia.lightclient.xyz --beacon.checkpoint 0x0014732c89a02315d2ada0ed2f63b32ecb8d08751c01bea39011b31ad9ecee36
+```
+
+### Running `blsync` as a Standalone Tool
+
+As mentioned before, `blsync` can be run in standalone mode. This will be similar to running a consensus client with low resource requirements and faster sync times. In most cases Geth users can use the integrated mode for convenience. The standalone mode can be used e.g. to drive an execution client other than Geth.
+
+#### Installing
+
+Depending on your [installation method](/docs/getting-started/installing-geth) either you have access to the `blsync` binary or you will have to build it from source by:
+
+```terminal
+go build ./cmd/blsync
+```
+
+#### Running
+
+Blsync takes the same flags as above to configure the HTTP endpoint as well as checkpoint. It additionally needs flags to connect to the execution client. Specifically `--blsync.engine.api` to configure the Engine API url and `--blsync.jwtsecret` for the JWT authentication token.
+
+Again to sync the Sepolia network in this mode, first run Geth:
+
+```terminal
+./build/bin/geth --sepolia --datadir light-sepolia-dir
+```
+
+The logs will indicate the Engine API path which is by default `http://localhost:8551` and the path to the JWT secret created which is in this instance `./light-sepolia-dir/geth/jwtsecret`. Now blsync can be run:
+
+```terminal
+./blsync --sepolia --beacon.api https://sepolia.lightclient.xyz --beacon.checkpoint 0x0014732c89a02315d2ada0ed2f63b32ecb8d08751c01bea39011b31ad9ecee36 --blsync.engine.api http://localhost:8551 --blsync.jwtsecret light-sepolia-dir/geth/jwtsecret
+
+INFO [06-23|15:06:33.388] Loaded JWT secret file                   path=light-sepolia-dir/geth/jwtsecret crc32=0x5a92678
+INFO [06-23|15:06:34.130] Successful NewPayload                    number=6,169,314 hash=d4204e..772e65 status=SYNCING
+INFO [06-23|15:06:34.130] Successful ForkchoiceUpdated             head=d4204e..772e65 status=SYNCING
+```
+
+## Summary
+
+This guide should provide the steps of using blsync in both integrated and standalone modes with Geth as an alternative to running a full consensus client. This is suitable for non-production use-cases and offers lower resource requirements and higher speed. The steps to run blsync include choosing an endpoint, obtaining a checkpoint, and starting the client with correct flags.
```

Comment:
This whole paragraph is useless. ChatGPT just loves to repeat itself I guess.

Reply:
Haha I just tried to follow the structure of other pages, so explicitly asked chat to write a summary :)

Reply:
I have removed it. The page isn't very long and IMHO doesn't need a summary. Also sounds too much like AI.